waveorder/plugin/tab_recon.py

```diff
@@ -2656,9 +2631,7 @@ def get_pydantic_kwargs(
         for field, field_def in pydantic_model.__fields__.items():
             if field_def is not None and field not in excludes:
                 ftype = field_def.type_
-                if isinstance(ftype, BaseModel) or isinstance(
-                    ftype, ModelMetaclass
-                ):
+                if isinstance(ftype, BaseModel):
```

Comment:
Are these changes needed?

Reply:
In v2 this is not supported https://github.com/pydantic/pydantic/issues/6381. It's available in _some_ versions from a internal module but I don't understand this use case enough.

Reply:
And trying to import a non-existent symbol will prevent the entire plugin from starting.

Reply:
Hmm so it is available from the v1 namespace despite what maintainers said on that issue... reverted in https://github.com/mehta-lab/waveorder/pull/448/commits/b50820a59a6ad0d6dcc8ccda5134ccf77009c8d2

Reply:
I still can't get the recon tab to work though:

1. use a random HCS store that has a 'BF' channel as both input and output
2. click 'Load' and choose the example phase config
3. `KeyError: 'birefringence'`

Reply:
Sounds good. I'll tag @amitabhverma here so he can see this, and let's move ahead with this merge for now.

Thanks @ziw-liu.